src/ZEO/runzeo.py

```diff
@@ -342,6 +343,7 @@ def create_server(storages, options):
         storages,
         read_only = options.read_only,
         client_conflict_resolution=options.client_conflict_resolution,
+        msgpack=options.msgpack or os.environ.get('ZEO_MSGPACK'),
```

Comment:
So even if the user specifies `--msgpack=0` at the command line (assuming that's how the argument processing works), the environment variable will override? That seems backwards to me (explicit arguments should have precedence over implicit inherited environment; the other way around is hard to debug). Maybe have the default value of the argument be None, and only if the argument isn't specified at all look at the environment.